src/acquisition/covidcast/generate_islatest_fix_sql.py

```diff
@@ -1,7 +1,10 @@
+from delphi.epidata.acquisition.covidcast.logger import get_structured_logger
```

Comment:
@melange396 should this file be using the logger at all? it looks like it's meant to pipe stdout to a file.

Reply:
youre right, it is used with a pipe.  no logger needed here.